frontend-react/src/components/Admin/AdminFormEdit.tsx

```diff
@@ -9,7 +9,7 @@ import {
 export const TextInputComponent = (params: {
     fieldname: string;
     label: string;
-    defaultvalue: string;
+    defaultvalue: null | string;
```

Comment:
I think the "standard" is `string | null`

Reply:
fixed